### DISCUSSION TITLE
Can the software specify the output file name

### DISCUSSION THREAD

**Discussion by Achuan-2** [kblin/ncbi-genome-download#190] [Q&A]

**Comment by kblin** [comment]
`ncbi-genome-download` is designed to handle bulk downloads from the NCBI servers, so being able to specify output file names isn't really part of the feature set. However, if you use the `--flat-output` and `--output-folder` parameters when downloading only a single file, it should be straightforward to rename that one downloaded file afterwards. Like this:
```bash
ncbi-genome-download --assembly-accessions GCF_008931305.1 --flat-output --output-folder example bacteria
mv example/*.gz example/streptomyces_coelicolor.gbff.gz
```
Note that this will fail if you download more than one file.
